scripts/delete-unverified-subscribers.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const DB = require("../db/DB");
+
+
+async function deleteUnverifiedSubscribers() {
+  await DB.deleteUnverifiedSubscribers();
+}
+
+
+(async () => {
+  await deleteUnverifiedSubscribers();
+  process.exit();
```

Comment:
Does this need a `try..catch` in case something goes wrong? I’m fearing the `process.exit()` never executing if the delete errors and having a stalled child process.